pkg/templateservicebroker/servicebroker/deprovision.go

```diff
@@ -27,6 +27,7 @@ func (b *Broker) Deprovision(u user.Info, instanceID string) *api.Response {
 		if kerrors.IsNotFound(err) {
 			return api.NewResponse(http.StatusGone, &api.DeprovisionResponse{}, nil)
 		}
+		glog.Errorf("Failed BrokerTemplateInstances.Get(): instanceID %s", instanceID)
```

Comment:
We should be logging these at a higher level, not in these methods.  If the broker API needs generic logging, it should have that at an appropriate level.

Reply:
understood, I will remove/restore the added loggings